pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -234,13 +236,57 @@ public static Profile addParticipantProfileFromFile(String esIndex, String fileN
         }
     }
 
+    public static Dsm addDsmObjectToParticipantFromFile(String esIndex, String fileName, String ddpParticipantId, String dob) {
+        Gson gson = new Gson();
+        String json = null;
+        try {
+            json = TestUtil.readFile(fileName);
+            json = json.replace("<dateOfBirth>", dob);
```

Comment:
Do you want to catch an exception in json.replace?  Or is it really just readFile() that needs exception handling?

Reply:
just the readFile, it's only in tests and most probably will just happen if the tests are running with wrong env variables